awxkit/awxkit/cli/options.py

```diff
@@ -94,9 +96,13 @@ def __init__(self, v2, page, resource, parser):
         self.handle_custom_actions()
 
     def get_allowed_options(self):
-        self.allowed_options = self.page.connection.options(
-            self.page.endpoint + '1'
-        ).headers.get('Allow', '').split(', ')
+        options = self.page.connection.options(
+            self.page.endpoint + '1/'
+        )
+        warning = options.headers.get('Warning', '')
```

Comment:
I think this is a good pattern overall.

Couldn't you just keep it as a warning, and not try to interpret it as a deprecation warning? Like, the AWX CLI doesn't really need to know that the warning is about deprecation. If there's a warning, might as well print it.

Reply:
Yea, I could see the argument for that.

Reply:
As a correspondence to the code & agent, that's a super standard approach.